array-manipulation-3/src/difference.js

```diff
@@ -0,0 +1,15 @@
+/* exported difference */
+function difference(first, second) {
+   var firstDiff= first.filter(function(element) {
+       if(!second.includes(element)) {
+           return element
+       }
+   });
+  var secondDiff= second.filter(function(element) {
+       if(!first.includes(element)) {
+           return element
+       }
+   });
```

Comment:
Try doing this with loops instead of `.filter`. `.filter()` doesn't work the way you think it does.